modules/ROOT/pages/drools-module-reference.adoc

```diff
@@ -71,7 +71,7 @@ Syntax:
 
 [NOTE]
 ====
-The Drools libraries are bundled with the Mule distribution. Drools 5.0 is the latest supported version.
+Starting in Mule 3.9.1, Drools libraries are no longer bundled with the Mule distribution. Drools 5.0 is the latest supported version. To add the Drools module along with its libraries to 3.9.1 or later, add the Maven dependency to your Mule application's `pom.xml` file. See <<maven, Maven>>. 
```

Comment:
You do have Mule as Mule runtime engine (Mule) at the first occurrence on this page, right?